features/admin.applications.v1/pages/applications-settings.tsx

```diff
@@ -0,0 +1,430 @@
+            setAuthenticationRequired(dcrConfigs?.authenticationRequired);
+            setEnableFapiEnforcement(dcrConfigs?.enableFapiEnforcement);
+            setSsaJwks(dcrConfigs?.ssaJwks);
+        }
+    }, [ dcrConfigs ]);
+
+    useEffect(() => {
+        if (dcrEndpoint !== undefined) {
+            return;
+        }
+
+        // Fetch the server endpoints for OIDC applications.
+        getOIDCApplicationConfigurations()
+            .then((response) => {
+                setDCREndpoint(response.dynamicClientRegistrationEndpoint);
+            })
+            .catch(() => {
+                dispatch(addAlert({
+                    description: t("applications:notifications.fetchOIDCIDPConfigs.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("applications:notifications.fetchOIDCIDPConfigs.genericError.message")
+                }));
+            });
+    });
+
+    const handleUpdateSuccess = () => {
+        dispatch(
+            addAlert({
+                description: t("jwtPrivateKeyConfiguration:notifications.success.description"),
+                level: AlertLevels.SUCCESS,
+                message: t(
+                    "console:develop.pages.applicationsSettings.notifications.success.message"
+                )
+            })
+        );
+    };
+
+    const resolveConnectorUpdateErrorMessage = (error: AxiosError): string => {
+        
+        return (
+            t("console:develop.pages.applicationsSettings.notifications.error.description",
+                { description: error.response.data.description })
+        );
+    };
+
+    const handleUpdateError = (error: AxiosError) => {
+        if (error.response && error.response.data && error.response.data.detail) {
+            dispatch(
+                addAlert({
+                    description: resolveConnectorUpdateErrorMessage(error),
+                    level: AlertLevels.ERROR,
+                    message: t(
+                        "console:develop.pages.applicationsSettings.notifications.error.message"
+                    )
+                })
+            );
+        } else {
+            // Generic error message
+            dispatch(
+                addAlert({
+                    description: t(
+                        "console:develop.pages.applicationsSettings.notifications.genericError.description"
+                    ),
+                    level: AlertLevels.ERROR,
+                    message: t(
+                        "console:develop.pages.applicationsSettings.notifications.genericError.message"
+                    )
+                })
+            );
+        }
+    };
+
+    /**
+     * Prepare form values for submitting.
+     *
+     * @param values - Form values.
+     * @returns Sanitized form values.
+     */
+    const updateConfigurations = (values: ApplicationsSettingsFormValuesInterface) => {
+        setIsSubmitting(true);
+        const updateData: any = [
```

Comment:
Please consider using a proper type definition here rather than `any`
```suggestion
        const updateData: any = [
```

Reply:
Fixed with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)